plugins/main/public/controllers/agent/index.js

```diff
@@ -9,25 +9,23 @@
  *
  * Find more information about this on the LICENSE file.
  */
-import { AgentsController } from './agents';
 import { RegisterAgent } from '../../components/endpoints-summary/register-agent/containers/register-agent/register-agent';
-import { ExportConfiguration } from './components/export-configuration';
-import { AgentsWelcome } from '../../components/common/welcome/agents-welcome';
+import { ExportConfiguration } from '../../components/agents/export-configuration';
 import { Mitre } from '../../components/overview';
 import { AgentsTable } from '../../components/endpoints-summary/table/agents-table';
 import { MainModule } from '../../components/common/modules/main';
 import { MainSyscollector } from '../../components/agents/syscollector/main';
 import { MainAgentStats } from '../../components/agents/stats';
 import { getAngularModule } from '../../kibana-services';
 import { MainEndpointsSummary } from '../../components/endpoints-summary';
+import { AgentView } from '../../components/endpoints-summary/agent';
 
 const app = getAngularModule();
 
 app
-  .controller('agentsController', AgentsController)
   .value('RegisterAgent', RegisterAgent)
   .value('ExportConfiguration', ExportConfiguration)
```

Comment:
**suggestion**: this seems not to be used, we could remove it.

Reply:
I removed the entries for ExportConfiguration, AgentsWelcome, Mitre, AgentsTable, MainSyscollector and MainAgentStats since they do not appear in any HTML templates.